web-app/django/VIM/apps/instruments/management/commands/import_instruments.py

```diff
@@ -125,8 +135,10 @@ def handle(self, *args, **options) -> None:
                     for ins in instrument_list[ins_i : ins_i + 50]
                 ]
                 ins_data: list[dict] = self.get_instrument_data(ins_ids_subset)
-                ins_imgs_subset: list[str] = [
-                    ins["image"] for ins in instrument_list[ins_i : ins_i + 50]
-                ]
-                for instrument_attrs, ins_img_url in zip(ins_data, ins_imgs_subset):
-                    self.create_database_objects(instrument_attrs, ins_img_url)
+                for instrument_attrs, ins_id in zip(ins_data, ins_ids_subset):
+                    img_dir = "../../static/instruments/images/instrument_imgs"
+                    original_img_path = f"{img_dir}/original/{ins_id}.png"
+                    thumbnail_img_path = f"{img_dir}/thumbnail/{ins_id}.png"
+                    self.create_database_objects(
+                        instrument_attrs, original_img_path, thumbnail_img_path
+                    )
```

Comment:
I feel skeptical about these URLs and am not sure that your browser will be able to find the images with these URLs. They are being served when you run this locally? At the very least I think we should make these absolute paths.